code/__HELPERS/mobs.dm

```diff
@@ -329,6 +329,11 @@ Proc for attack log creation, because really why not
 
 	return FALSE
 
+/proc/is_superior_animal(var/mob/M)
+	if(istype(M, /mob/living/carbon/superior_animal))
+		return TRUE
+	return FALSE
+
```

Comment:
```suggestion
```
There is already a superior animal check in is_helpers.dm.

Reply:
I missed it lol